account-kit/infra/src/client/smartAccountClient.ts

```diff
@@ -174,21 +171,26 @@ export function createAlchemySmartAccountClient(
       ? alchemyUserOperationSimulator(config.transport)
       : undefined,
     signUserOperation: config.signUserOperation,
+    // @ts-expect-error
+    addBreadcrumb(crumb: string) {
+      const oldConfig = config.transport.config;
+      const dynamicFetchOptions = config.transport.dynamicFetchOptions;
+      const newTransport = alchemy({ ...oldConfig });
```

Comment:
so when we do this, dynamic fetch options is reset to what the original fetch options were in the oldConfig (passed in by the user)